website/versioned_docs/version-v1.x/installation.md

```diff
@@ -12,10 +12,10 @@ sidebar_position: 3
 You need the following to use MACI:
 
 - Node.js: use [`nvm`](https://github.com/nvm-sh/nvm) to install it. MACI has
-  been tested with Node 14, 16 and 18. We do however recommend to use Node 18 as Node 14 is deprecated and Node 16 will soon be deprecated too.
+  been tested with Node 14, 16, 18 and 20. We do however recommend to use Node 18 or Nove 20 as Node 14 is deprecated and Node 16 will soon be deprecated too.
```

Comment:
```suggestion
  been tested with Node 14, 16, 18 and 20. We do however recommend to use Node 18 or Node 20 as Node 14 is deprecated and Node 16 will soon be deprecated too.
```

Reply:
Sidenote, this doesn't seem like a change that should be part of this PR 😄

Reply:
I'm ok with it, just noting for future PRs

Reply:
😆 true